turbinia/evidence.py

```diff
@@ -132,7 +136,12 @@ def serialize(self):
     """Return JSON serializable object."""
     serialized_evidence = self.__dict__
     if self.parent_evidence:
-      serialized_evidence['parent_evidence'] = self.parent_evidence.serialize()
+      if isinstance(self.parent_evidence, dict):
+        # The parent evidence may have been serialized already
+        serialized_evidence['parent_evidence'] = self.parent_evidence
+      else:
+        serialized_evidence['parent_evidence'] = self.parent_evidence.serialize(
```

Comment:
Heh, is this really the way YAPF formats this line?  Maybe start the right-hand side of the assignment with a paren so the rest of the right-hand side goes on the second line?